### PR TITLE
fix: handle time termination with "lessThan" condition

### LINKED ISSUE #8184: Market proposals are  not enacting or failing when oracle termination condition is set to LessThan (but time is > enactment)
When an oracle is used with a  term condition “Less Than” and a timestamp set to be after the enactment time the market state never moves to enacted/terminated/failed etc it just freezes at Passed.

Expecting the market to terminate immediately voting passes.

### PR DESCRIPTION
**Pull request by AnExsomnis**
close: #8184

### CODE REVIEW

File: core/governance/market.go [hunk 1]
```diff
@@ -264,6 +267,10 @@ func validateFuture(future *types.FutureProduct, decimals uint64, assets Assets,
 						return types.ProposalErrorInvalidFutureProduct, err
 					}
 
+					if cond.Operator == datapb.Condition_OPERATOR_LESS_THAN {
```
Comment (ze97286) [inline core/governance/market.go:270]: shouldn't this also be LTE?

Comment (AnExsomnis) [reply core/governance/market.go:270]: Yes, I think it should be

File: core/governance/market.go [hunk 2]
```diff
@@ -493,8 +500,17 @@ func validateUpdateFuture(future *types.UpdateFutureProduct, et *enactmentTime)
...
```
Comment (ze97286) [inline core/governance/market.go:503]: LTE too.

File: core/governance/market.go (hunk 1 above)
Comment (AnExsomnis) [reply core/governance/market.go]: This has been corrected, thanks!

File: core/governance/engine.go [hunk 3]
```diff
@@ -998,7 +997,11 @@ func (e *Engine) updatedMarketFromProposal(p *proposal) (*types.Market, types.Pr
 		return nil, types.ProposalErrorUnsupportedProduct, ErrUnsupportedProduct
 	}
 
-	if perr, err := validateUpdateMarketChange(terms, &enactmentTime{current: p.Terms.EnactmentTimestamp}); err != nil {
+	if perr, err := validateUpdateMarketChange(
+		terms,
+		&enactmentTime{current: p.Terms.EnactmentTimestamp},
+		e.timeService.GetTimeNow().Unix(),
```
Comment (ze97286) [inline core/governance/engine.go:1003]: shouldn't this be `UnixNano()`?

**Review by EVODelavega — DISMISSED** [review]

**Review by EVODelavega — DISMISSED** [review]

**Review by ze97286 — APPROVED** [review]

**Review by EVODelavega — APPROVED** [review]

**Pull request merged**